spec/requests/oauth/authorization_grants_controller_spec.rb

```diff
@@ -24,34 +24,35 @@
   describe 'POST /create' do
     subject(:call_endpoint) { post oauth_authorization_grants_path, params: { state:, approve: } }
 
-    let(:user) { create(:user) }
+    let_it_be(:user) { create(:user) }
```

Comment:
What is the difference between `let` and `let_it_be`?

Reply:
So `let_it_be` is a helper that `test-prof` provides. It helps avoid recreating a record between scenarios when that record doesn't change.

In this case, `user` doesn't need to be different between the scenarios in the spec, so `let_it_be` instantiates and persists it between runs. This results in only 1 user record being created instead of 4 (3 scenarios + 1 shared example).

Reply:
Here's a link to the `test-prof` documentation for `let_it_be` in case you want to dig deeper: https://test-prof.evilmartians.io/recipes/let_it_be